doc/modules/cassandra/pages/developing/cql/functions.adoc

```diff
@@ -92,9 +92,9 @@ into its own datatype.
 The conversions rely strictly on Java's semantics. For example, the
 double value 1 will be converted to the text value '1.0'. For instance:
 
-[source,cql]
+[source,sql]
```

Comment:
Was this `cql` to `sql` intended?

Reply:
@bbotella intentional because AsciiDoc doesn't have a 'cql' formatter, and sql works pretty well.